go/libkb/sig_chain.go

```diff
@@ -712,7 +712,7 @@ func reverseListOfChainLinks(arr []ChainLinks) {
 	}
 }
 
-func (c ChainLinks) popNRightmostLinks(n int) ChainLinks {
+func (c ChainLinks) omittingNRightmostLinks(n int) ChainLinks {
```

Comment:
You know I remembered talking about this, but I totally forgot what we said. I'll drop this commit :)